plugins/ledger/ledger.py

```diff
@@ -371,7 +371,11 @@ def sign_transaction(self, tx, password):
             if segwitTransaction:
                 self.get_client().startUntrustedTransaction(True, inputIndex,
                                                             chipInputs, redeemScripts[inputIndex])
-                outputData = self.get_client().finalizeInputFull(txOutput)
+                if changePath:
+                    outputData = self.get_client().finalizeInput(output, format_satoshis_plain(outputAmount),
```

Comment:
`output` is `str` here but `btchip-python` expects `bytes`. You can import `to_bytes` from `electrum.util` to convert it. Not sure if it's better to convert here or where its value gets set initially.